server.js

```diff
@@ -1,27 +1,66 @@
-import express from "express";
-import cors from "cors";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+import express from "express"; 
+import cors from "cors"; 
+import goldenGlobesData from "./data/golden-globes.json";
+
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors());
-app.use(express.json());
+app.use(express.json()); 
 
-// Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send({responseMessage: "Golden Globe stats, library at /globes"}); 
```

Comment:
I used this first path to give some more information of the different endpoints. I think it would be nice to implement it. You could try with something similar to this: 

```
app.get("/", (req, res) => {
  res.status(200).json({
    resMessage: "Netflix movies",
    endPointsArray: "/movies - Returns an array of all movies",
    endPointsArrayQuery: "/movies ? type, director, year",
    endPointsSingleItem: "/movies/id/:id - /movies/title/:title"
  });
});

```

Reply:
Hej David, thanks for taking your time to review and helping me out with your great tips! Much appreciated 🫶